apps/src/tutorialExplorer/image.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Simple img replacement that fades in upon load.
+ * Based on http://buildwithreact.com/article/fade-in-image-recipe
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const Image = React.createClass({
+  getInitialState: function () {
+    return {
+      loaded: false
+    };
+  },
+
+  onImageLoad: function () {
+    this.setState({ loaded: true });
+  },
+
+  componentDidMount: function () {
+    const imgTag = ReactDOM.findDOMNode(this.refs.img);
+    const imgSrc = imgTag.getAttribute('src');
+    const img = new window.Image();
+    img.onload = this.onImageLoad;
+    img.src = imgSrc;
+  },
+
+  render: function () {
+    const {...props} = this.props;
```

Comment:
Doesn't seem like there's a lot of value in doing this instead of just having line 32 be {...this.props}